compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/parser/BLangPackageBuilder.java

```diff
@@ -3123,4 +3128,110 @@ void endMatchExpression(DiagnosticPos pos, Set<Whitespace> ws) {
         matchExpr.addWS(ws);
         addExpressionNode(matchExpr);
     }
+
+    public BLangFunction getScopesFunctionDef(DiagnosticPos pos,
+            Set<Whitespace> ws,
+            boolean publicFunc,
+            boolean nativeFunc,
+            boolean bodyExists,
+            boolean isReceiverAttached, String name) {
```

Comment:
Unused variable isReceiverAttached? and publicFunc and nativeFunc always have constant values